bocas/artifacts/yaml_utils.py

```diff
@@ -0,0 +1,20 @@
+import yaml
+
+
+def parse_yaml_node(loader, node):
```

Comment:
can this support custom artifacts?

Reply:
Yes, so long that the key/value pairs are registered objects.  To make new artifact types, `yaml_tag` and `to_yaml()` must be defined along with registering the types in `yamlify.py`.  This function should be generic enough to parse most yaml strings